src/utils/selectedCellUtils.ts

```diff
@@ -119,12 +137,58 @@ export function getNextSelectedCellPosition<R, SR>({
     }
   };
 
+  const getParentRowIdx = (parent: CalculatedColumnParent<R, SR>) => {
+    return parent.level + mainHeaderRowIdx;
+  };
+
+  const setHeaderGroupColAndRowSpan = (moveRight: boolean) => {
```

Comment:
Is `moveRight` always about moving right? What about right-to-left layouts? Should this be `moveNext` instead?

Reply:
good point